### ISSUE TITLE
html anchor destinations are hidden by the top navigation bar

### ISSUE THREAD

**Issue by dmsimard** [patternfly/patternfly-3#352]
When using the [vertical navigation](https://www.patternfly.org/pattern-library/navigation/vertical-navigation/) and html in-page anchors (i.e, href="#anchor" and an item with id="anchor"), the destination of the anchor is hidden by the top navigation bar.

If, for example, I have the following:

```
<h2 id="Utilization" class="card-pf-title" style="">
    <a href="#Utilization">Utilization</a>
</h2>
```

When clicking on "Utilization", this is what I would expect:

![expectation](http://i.imgur.com/l2TKYJj.png)

While, in reality, this is what happens:

![reality](http://i.imgur.com/u1CRDun.png)

I have worked around this issue to some extent in CSS but it is not a good solution:

```
a:before {
  display: block;
  content: " ";
  margin-top: -64px;
  height: 64px;
  visibility: hidden;
}
```

This has for a result to push the anchor further down but it also causes the links to have bigger boundaries and be clickable outside of the text which is a problem.

**Comment by andresgalante** [comment]
Thanks @dmsimard I'll try to find the less hacky way to solve this.
@LHinson can we open a story to look into this please?

**Comment by dmsimard** [comment]
@andresgalante would love to hear if you have had the time to try anything out yet. Thanks.

**Comment by LHinson** [comment]
Hey @dmsimard, just wanted to let you know that @andresgalante is actually on paternity leave atm. He will be returning next week though! In the meantime, I've added a bug in our backlog to help us track this. https://patternfly.atlassian.net/browse/PTNFLY-1710

**Comment by dabeng** [comment]
@LHinson  I think  [this issue](https://github.com/patternfly/patternfly/issues/338) is the root cause.

**Comment by dmsimard** [comment]
@dabeng you think ? This issue is kind of frustrating for me. I like the patternfly navbars but I don't have this issue with pure bootstrap navbars :(

**Comment by dmsimard** [comment]
@LHinson @andresgalante I would love to know if you guys have found anything for this :(

Surely using link anchors with the vertical navigation is a fairly common use case ?

**Comment by dmsimard** [comment]
I ended up using http://stackoverflow.com/questions/10732690/offsetting-an-html-anchor-to-adjust-for-fixed-header/13067009#13067009 which seems to work fine so far.

**Comment by andresgalante** [comment]
Hi @dmsimard sorry for the very late reply. I don't think we can add a global position of anchors. The best solution I can think of is adding a spacer on the body when the header is fixed, I'll document that when creating templates with fixed header on the new version of patternfly.
